src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -5916,5 +5916,36 @@ public void GetSpecialType_ThrowsOnGreaterThanCount()
 
             Assert.True(exceptionThrown, $"{nameof(comp.GetSpecialType)} did not throw when it should have.");
         }
+
+        [Fact]
+        [WorkItem(34984, "https://github.com/dotnet/roslyn/issues/34984")]
+        public async Task ConversionIsExplicit_UnsetConversionKind()
+        {
+            var source =
+@"class C1
+{
+}
+
+class C2
+{
+    public void Foo() 
+    {
+        var c = new C1();
+        foreach (string item in c.Items)
+        {
+        }
+}";
+            var comp = CreateCompilation(source);
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var root = await tree.GetRootAsync();
```

Comment:
>await tree.GetRootAsync(); [](start = 23, length = 26)

No need to use async test method. Can just use `var root = tree.GetRoot();` here